docs/changes.rst

```diff
@@ -23,8 +26,13 @@ Released: not yet
 
 **Enhancements:**
 
+* Added support for managing WBEM indication listeners by adding a command
```

Comment:
We should also state that we moved a number of utility functions from pywbemcli to a _utils module.

Reply:
DONE - added a sentence to the existing item in the Cleanup section. Also, since this is supposed to just be the preparation, removed the entry about the listener in the Enhancements section.